.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
```

Comment:
We don't need this here

Reply:
We sure need it here, because otherwise lintsraged doesn't work. It works only like this correctly.

Reply:
I see that everything works
![image](https://github.com/user-attachments/assets/aa191e66-b992-4c8d-882b-e2195a1ac889)

Reply:
I misunderstood the comment and thought that the whole file had to be removed.
Regardless, I disagree that we do not need this:

``` bash
#!/bin/sh
. "$(dirname "$0")/_/husky.sh"
```

But I accept that husky runs this somehow, even without this header. 

Changes were incorporated in the file as you asked.
https://github.com/rolling-scopes/site/pull/432/commits/5d60be569ee35833d5a4abc0aa5524c5ba581548